recipes-core/underclock/underclock/underclock

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Set to interactive to make PowerHAL function.
+echo interactive > /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor
+echo 0        > /sys/devices/system/cpu/cpu1/online
+echo 0        > /sys/devices/system/cpu/cpu2/online
+echo 0        > /sys/devices/system/cpu/cpu3/online
+
+echo 1 > /sys/power/memc_dfs_init
+
+# Fixup permissions so that the PowerHAL can actually change them.
+# android-init runs very early in the boot process after which some of these change. This means that we have to change the permissions later in the boot process.
+chown ceres:root /sys/class/input/input1/mode
```

Comment:
it's a bit wacky but on other watches we tend to do these things in the android init script, it makes more sense than in the underclock. underclock is meant to run at the end of the boot so that the boot is as fast as possible

Reply:
MagneFire informed me that I need to change my glasses. Given l11 and l12, this makes sense. Let's just ship it